kaggle_tools_update.Dockerfile

```diff
@@ -9,23 +9,12 @@
 # is enabled.
 
 # Usage:
-#   docker rmi gcr.io/kaggle-images/python:latest
+#   docker rmi gcr.io/kaggle-images/python:pinned
```

Comment:
You should add instructions on how to pin the current stable image

Reply:
Done.